core/src/test/java/dk/alexandra/fresco/lib/math/bool/add/TestFullAdder.java

```diff
@@ -0,0 +1,27 @@
+package dk.alexandra.fresco.lib.math.bool.add;
+
+import java.util.ArrayList;
+import java.util.List;
+import dk.alexandra.fresco.framework.value.SBool;
+import dk.alexandra.fresco.framework.DRes;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class TestFullAdder {
```

Comment:
Same for this name. Not sure what the best approach is, but a longer class name is needed I guess